tests/beaker_tests/cisco_bgp/test_bgp_non_defaults.rb

```diff
@@ -116,6 +113,13 @@
   'timer_bgp_holdtime'                     => '110',
 }
 
+expected_values_diff = {
```

Comment:
Instead of this approach of creating an entirely separate hash, consider modifying the contents of `expected_values` based on the platform. This is what we're already doing in feature/grpc for Nexus/XR differences - see https://github.com/cisco/cisco-network-puppet-module/blob/feature/grpc/tests/beaker_tests/bgp/test_bgp_non_defaults.rb#L101 for examples. If you can follow this approach now it will make the subsequent merge a lot less painful.